apps/condo/domains/common/components/TrackingContext.tsx

```diff
@@ -229,6 +229,7 @@ const TrackingProvider: React.FC = ({ children }) => {
             if (link) {
                 trackingProviderValueRef.current.userProperties['role'] = get(link, 'role.name')
                 trackingProviderValueRef.current.userProperties['organization'] = get(link, 'organization.name')
```

Comment:
I know it's too late, but
```suggestion
                trackingProviderValueRef.current.userProperties['organizationName'] = get(link, 'organization.name')
```

Reply:
Yeah, originally was no plan to send id there. So change key right now will break all of the created reports